src/utils/populate.js

```diff
@@ -25,6 +25,24 @@ export const getPopulateObj = (str) => {
 }
 /**
  * @private
+ * @description Determine the structure of the child parameter to populate onto
+ * @param {String|Object} child - Value at child parameter
+ */
+export const getChildType = (child) => {
```

Comment:
This is great for now, but eventually we might want to make this more general (return type of all types of vars) to be used elsewhere. Wish there was a lodash function to just return a variable's type [but it seems like they feel as though developers should make this themselves](https://github.com/lodash/lodash/issues/391).